lib/aussieaddonscommon/utils.py

```diff
@@ -211,20 +211,26 @@ def get_kodi_build():
     try:
         return xbmc.getInfoLabel("System.BuildVersion")
     except Exception:
-        return
+        return ''
```

Comment:
Returning None is usually best if you're going to later test it with `if build:` for example. As you've done here, an empty string also evaluates to False. Does this necessarily need to be a string?

Reply:
You're right - the changes in get_kodi_version make this redundant, I'll make it None.